style.css

```diff
@@ -1,250 +1,517 @@
-margin-bottom: 40px;
-margin-top: 135px;
+  text-align: center;
+  height: 10px;
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 10px;
+  color: #000;
+  margin-right: 142px;
+  margin-bottom: 26px;
 }
 
 .top__subtitle {
-display: flex;
-align-items: center;
-width: 250px;
-height: 16px;
-left: 600px;
-font-weight: normal;
-font-size: 15px;
-line-height: 18px;
-color: #000000;
-margin-left: 590px;
-margin-top: 43px;
-}
-
-.top__avto {
-display: flex;
-align-items: center;
-width: 968px;
-height: 450px;
-margin-left: 190px;
-margin-top: -323px;
+  text-align: center;
+  height: 16px;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 18px;
+  color: #000;
+  margin-right: 142px;
 }
 
 .top__fon {
-display: flex;
-align-items: center;
-width: 1514px;
-height: 467px;
-background: url("./images/fon.png");
-margin-top: -131px;
-margin-left: -10px;
+ position: inherit;
+ margin-bottom: 55px;
 }
 
-.middle__title {
-color: red;
-font-weight: 600;
-font-size: 90px;
-line-height: 10px;
-margin-left: 100px;
-margin-top:100px;
+.top__avto{
+  max-width: 100%;
 }
+
 .other {
-display: flex;
-justify-content: space-around;
-flex-flow: row nowrap;
-width: 1000px;
-height: 40px;
-margin-left: 200px;
+  display: flex;
+  justify-content: space-around;
+  margin: 0 auto;
+  max-width: 100%;
+  padding-right: 328px;
+}
+
+.other__block1 {
+  display: inline-block;
+
+}
+
+.other__speedometr {
+ position: inherit;
+}
+
+.other__speed {
+  position: absolute;
+  width: 56px;
+  height: 44.13px;
+}
+
+.other__text-left {
+  display: flex;
+  flex-direction: column;
+}
+
+.other__left {
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.other__sec {
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.other__bottom-l {
+  position: relative;
+  font-size: 12px;
+  line-height: 15px;
+  bottom: 17px;
+}
+
+.other__center {
+  position: inherit;
+}
+
+.other__road {
+  position: absolute;
+}
+
+.other__next-center {
+  display: flex;
+  flex-direction: column;
+}
+
+.other__text-center {
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.other__km {
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.other__bottom-с {
+  position: relative;
+  font-size: 12px;
+  line-height: 15px;
+  bottom: 17px;
+}
+
+.other__icon {
+ position: absolute;
+}
+
+.other__link_testdrive {
+  position: relative;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000000;
+  right: 40px;
+}
+
+.other__icon-speed {
+  position: relative;
+  bottom: 5px;
+  width: 22px;
+  height: 22px;
 }
 
 .other__click {
-display: flex;
-align-items: center;
-width: 160px;
-height: 40px;
-border: 2px solid #E51937;
-box-sizing: border-box;
-color: #E51937;
-background: #ffffff;
-margin-left: -100px;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-align: center;
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  box-sizing: border-box;
+  color: #e51937;
+  background: #fff;
+  margin-bottom: 15px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)